### ISSUE TITLE
Incomplete documentation around FluxComponent

### ISSUE THREAD

**Issue by chadly** [acdlite/flummox#208]
I spent a good hour trying to figure this out until I found #61. `flux` it is not available on the react context unless you explicitly include it in `contextTypes`:

```js
contextTypes: {
    flux: React.PropTypes.any
}
```

`this.context.flux` is undefined in a react component until you add the `contextType`. Just adding a note to that effect to the docs would be enormously helpful to beginners.

**Comment by acdlite** [comment]
That's a good point. Docs assume some knowledge of context already, which isn't nice for beginners.

**Comment by burabure** [comment]
I explain context in my updated docs.#136 #169

**Comment by pburtchaell** [comment]
Yeah, I don't think it is good to assume knowledge of context since it is rather undocumented by the React team to begin with (which is reasonable considering it is "experimental"). :+1: to a PR to update the docs with that note.
